<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResizeDialog</class>
 <widget class="QDialog" name="ResizeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Resize</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Size</string>
     </property>
     <layout class="QGridLayout" name="_6">
      <item row="0" column="1">
       <widget class="QSpinBox" name="widthSpinBox">
        <property name="suffix">
         <string> tiles</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>99999</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Width:</string>
        </property>
        <property name="buddy">
         <cstring>widthSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Height:</string>
        </property>
        <property name="buddy">
         <cstring>heightSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="heightSpinBox">
        <property name="suffix">
         <string> tiles</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>99999</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Offset</string>
     </property>
     <layout class="QGridLayout" name="_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>X:</string>
        </property>
        <property name="buddy">
         <cstring>offsetXSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="offsetXSpinBox">
        <property name="suffix">
         <string> tiles</string>
        </property>
        <property name="minimum">
         <number>-99999</number>
        </property>
        <property name="maximum">
         <number>99999</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Y:</string>
        </property>
        <property name="buddy">
         <cstring>offsetYSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="offsetYSpinBox">
        <property name="suffix">
         <string> tiles</string>
        </property>
        <property name="minimum">
         <number>-99999</number>
        </property>
        <property name="maximum">
         <number>99999</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="_5">
      <item>
       <widget class="Tiled::ResizeHelper" name="resizeHelper" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="removeObjectsCheckBox">
     <property name="text">
      <string>Remove objects outside of the map</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Tiled::ResizeHelper</class>
   <extends>QWidget</extends>
   <header>resizehelper.h</header>
   <container>1</container>
   <slots>
    <signal>offsetXChanged(int)</signal>
    <signal>offsetYChanged(int)</signal>
    <slot>setOffsetX(int)</slot>
    <slot>setOffsetY(int)</slot>
    <slot>setNewWidth(int)</slot>
    <slot>setNewHeight(int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>widthSpinBox</tabstop>
  <tabstop>heightSpinBox</tabstop>
  <tabstop>offsetXSpinBox</tabstop>
  <tabstop>offsetYSpinBox</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ResizeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>399</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>offsetXSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>resizeHelper</receiver>
   <slot>setOffsetX(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>285</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>offsetYSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>resizeHelper</receiver>
   <slot>setOffsetY(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>285</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>widthSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>resizeHelper</receiver>
   <slot>setNewWidth(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>285</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>heightSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>resizeHelper</receiver>
   <slot>setNewHeight(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>285</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resizeHelper</sender>
   <signal>offsetXChanged(int)</signal>
   <receiver>offsetXSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resizeHelper</sender>
   <signal>offsetYChanged(int)</signal>
   <receiver>offsetYSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ResizeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>385</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>353</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
